package.json

```diff
@@ -15,10 +15,8 @@
   "description": "Connect allthethings",
   "devDependencies": {
     "autoprefixer": "^6.3.3",
-    "babel": "5.8.12",
-    "babel-core": "5.8.12",
-    "babel-eslint": "^5.0.0",
-    "babel-loader": "5.3.2",
+    "babel-eslint": "^6.0.3",
+    "babel-plugin-add-module-exports": "^0.1.3",
```

Comment:
We use require in a couple spots
On May 2, 2016 8:54 AM, "Andrew Duthie" <notifications@github.com> wrote:

> In package.json
> <https://github.com/Automattic/woocommerce-connect-client/pull/249#discussion_r61758238>
> :
>
> > @@ -15,10 +15,8 @@
> >    "description": "Connect allthethings",
> >    "devDependencies": {
> >      "autoprefixer": "^6.3.3",
> > -    "babel": "5.8.12",
> > -    "babel-core": "5.8.12",
> > -    "babel-eslint": "^5.0.0",
> > -    "babel-loader": "5.3.2",
> > +    "babel-eslint": "^6.0.3",
> > +    "babel-plugin-add-module-exports": "^0.1.3",
>
> Assuming you're using ES6 import syntax consistently through the project,
> I don't think you need this plugin.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/Automattic/woocommerce-connect-client/pull/249/files/76f0a431f046bcfd25ecd35753cc04fa800a15b6#r61758238>
>